examples/sn-blog-gatsby/package.json

```diff
@@ -22,10 +22,16 @@
   },
   "dependencies": {
```

Comment:
gh-pages (not shown in changes) - will we publish it on github pages? what happens with the currently deployed site?

Reply:
gh-pages is not needed i guess